grammar.js

```diff
@@ -108,7 +108,7 @@ module.exports = grammar({
         ";",
       ),
 
-    enum_label: $ => seq(field("name", $.ident), optional(seq("=", $.integer))),
+    enum_label: $ => seq(field("name", $.ident), optional(seq("=", $.expression))),
```

Comment:
Thanks, I didn't even realize we supported chars for enum values.

Let's restrict to either integers or chars here.